source-kafka/README.md

```diff
@@ -0,0 +1,137 @@
+# `source-kafka` Connector
+
+This is an [Airbyte Specification-compatible](https://docs.airbyte.io/understanding-airbyte/airbyte-specification) connector that reads from Kafka topics/partitions.
+
+## Getting Started
+
+The easiest way to start using the connector is from Docker. The latest images are published at [Estuary's Package Registry](https://github.com/orgs/estuary/packages)
```

Comment:
It might be good to lead this with something like:

> To get started using this connector with Estuary Flow, run `flowctl discover --image ghcr.io/estuary/source-kafka:TAG`. The remainder of this README is geared toward developers who wish to contribute or integrate the connector in other projects.